config/schema/pubkey_encrypt.schema.yml

```diff
@@ -1,6 +1,6 @@
 pubkey_encrypt.initialization_settings:
```

Comment:
@nerdstein I think appending module name is already doing the job of making it non-generic. Don't you think something like pubkey_encrypt.pubkey_encrypt.initialization_settings would be too long?